tests_e2e/lisa/tests/agent_bvt/custom_script.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import uuid
+import sys
+
+from tests_e2e.scenario_utils.extensions.CustomScriptExtension import CustomScriptExtension
+
+
+def main(subscription_id, resource_group_name, vm_name):
+    os.environ["VMNAME"] = vm_name
+    os.environ['RGNAME'] = resource_group_name
+    os.environ["SUBID"] = subscription_id
+    os.environ["SCENARIONAME"] = "BVT"
+    os.environ["LOCATION"] = "westus2"
+    os.environ["ADMINUSERNAME"] = "somebody"
+    os.environ["BUILD_SOURCESDIRECTORY"] = "/somewhere"
+
+    cse = CustomScriptExtension(extension_name="testCSE")
+
+    ext_props = [
+        cse.get_ext_props(settings={'commandToExecute': f"echo \'Hello World! {uuid.uuid4()} \'"}),
+        cse.get_ext_props(settings={'commandToExecute': "echo \'Hello again\'"})
+    ]
+
+    cse.run(ext_props=ext_props)
```

Comment:
I noticed all of the logging from cse.run is being logged as lisa.stderr. Why is lisa capturing these as stderr instead of stdout?